### PR TITLE
Correctly load base64-encoded strings from Secret Yaml files.

### PR DESCRIPTION
**Pull request by johndaniels**
Loading secrets from YAML files was failing for me. Let me know if this change is the right way to fix that, thanks!

### CODE REVIEW

**Comment by johndaniels** [comment]
Should be all good on the contributor agreement.

File: util/src/main/java/io/kubernetes/client/util/Yaml.java [hunk 1]
```diff
@@ -19,11 +19,8 @@
 import java.io.IOException;
 import java.io.Reader;
 import java.io.StringReader;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```
Comment (brendandburns) [inline util/src/main/java/io/kubernetes/client/util/Yaml.java]: As a matter of style, we don't use wild-card imports.

File: util/src/test/java/io/kubernetes/client/util/YamlTest.java [hunk 2]
```diff
@@ -15,14 +15,12 @@
 import static org.junit.Assert.*;
 
 import com.google.common.io.Resources;
-import io.kubernetes.client.models.AppsV1beta1Deployment;
-import io.kubernetes.client.models.V1ObjectMeta;
-import io.kubernetes.client.models.V1Service;
-import io.kubernetes.client.models.V1ServicePort;
+import io.kubernetes.client.models.*;
```
Comment (brendandburns) [inline util/src/test/java/io/kubernetes/client/util/YamlTest.java]: as above wrt to wildcard imports.

**Comment by brendandburns** [comment]
I think this is ok, but I think that we need the encoding side of this also?  e.g. if I place bytes into a `V1Secret` object it should encode them too?

Basically, can you add a class that tries to round-trip a `V1Secret` if that test passes then I think we're good to go.

**Comment by johndaniels** [comment]
Still aware of this. I started on changes but didn't have time to finish them. It looks like round-tripping with YAML wasn't tested at all so it's a little bit more involved to get that working but I think it will be good.

**Comment by brendandburns** [comment]
@johndaniels thanks! sorry to send you off on a bigger change, but it will definitely help improve the code!

**Comment by johndaniels** [comment]
Ok, fixed the issue with the wildcard imports and got everything successfully round-tripping back to Yaml. Let me know if anything else needs changing.

**Comment by brendandburns** [comment]
Thanks, changes LGTM, merging.

**Pull request merged**